### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raproid**

### CODE REVIEW

**Comment by raproid** [comment]
Sofia, I've tested the code examples. They do work. 
We also discovered that cobra example was obsolete and didn't work, and Sergei replaced it with a working example that we, again, tested.

Please accept the PR, I need at least one approving review to merge it.

File: README.md [hunk 1]
```diff
@@ -31,17 +31,18 @@ Insolar is extensively documented. If you need more information on what is Insol
...
+Run the example:
 
-```
-go run ./example/example.go --config="./example/example_config.yaml"
+```go
+run ./example/example.go --config="./example/example_config.yaml"
 ```
 
-## In your code
+To test the code examples below, add them to `example.go` or test them directly in your code.
```
Comment (sofiayem) [inline README.md:42]: To test... test them directly.
Suggestion:
In addition, make amendments to the example as shown in code snippets below: add them to `example.go` or to your code directly.

Comment (raproid) [reply README.md:42]: What if the user doesn't wanna use the example.go?

Suggestion:
In addition, test the code snippets below replacing the `func main()` content in `example.go` with them. 
Modify the snippets appropriately to include them directly into your code.

Comment (raproid) [reply README.md:42]: No, won't do. Wait...

Comment (raproid) [reply README.md:42]: In addition, make amendments to `example.go` as shown in code snippets below to test the options. 
Modify the snippets appropriately to include them directly into your code.

Comment (raproid) [reply README.md:42]: We cannot predict the content of vars and config field in the "include into your code" option. Hence, the  vague "appropriately".

Comment (sofiayem) [reply README.md]: Then:
In addition:
* Test the code snippets below by replacing the `func main()` content in `example.go` with them.
* Modify the snippets appropriately to include them directly into your code—pay attention to variable and configuration field contents.

Comment (raproid) [reply README.md]: done

**Pull request merged**